starknet_py/contract_test.py

```diff
@@ -56,10 +56,10 @@
 
 # fmt: off
 
-EXPECTED_HASH = 0x92da7ab8dd6a7082182e4531fc6c886040e3176a3b6b78c353e5fe1267810a
-EXPECTED_HASH_WITH_IMPORTS = 0x66887e10960bd31234e20b6ee10b194dff8021db3a8ed3cc59496ac6c2d8d9
-EXPECTED_ADDRESS = 0x2d558d360480e06c63ef648dc19af68e7a57f80c0a500e6622019b17f5ba6f
-EXPECTED_ADDRESS_WITH_IMPORTS = 0x6f7481d6ed093f7cb8d944d9ff310bcd276e4b03e1b367ed8c0057a09f58208
+EXPECTED_HASH = 0x31e5eaebb50a32a6f6a1df5c2fe56a3568fe393249046bcec29df3371b0ed41
+EXPECTED_HASH_WITH_IMPORTS = 0x15c4b33c7ca117c4af320137fa837f5d4dc0c59142b9cf1cfde05cb539d0f21
+EXPECTED_ADDRESS = 0x26cbdbfba327039b9cf53825844af4af1d853d4846f7383859cfa44d4775664
+EXPECTED_ADDRESS_WITH_IMPORTS = 0x59c61849a74ea80ad8254dae4a1d7d8868cf12eceb903e57b8d0cb2d9e11294
```

Comment:
Cairo-lang got bumped from 0.11.0 to 0.11.0.1, compiler version changed and so the hashes.